### PR TITLE
Make player fall through dark matter

### PR DESCRIPTION
**Pull request by Anonymous4045**

### CODE REVIEW

**Review by hkohko — DISMISSED** [review]
falling works

**Review by hkohko — APPROVED** [review]
game runs now, falling works

File: pyweek36/constants.py [hunk 1]
```diff
@@ -67,7 +73,13 @@ class InputType(IntEnum):
...
+WALL_TEXTURE_PATH = ASSETS_DIR / "sprites/map/solid_block.png"
 PLAYER_IDLE_ANIM_PATH = ASSETS_DIR / "player/idle"
 PLAYER_JUMP_ANIM_PATH = ASSETS_DIR / "player/jump"
 PLAYER_WALK_ANIM_PATH = ASSETS_DIR / "player/walk"
 LOOPING_TEXTURES = {"idle", "walk"}
+
+# Load texture now so it isn't loaded from disk every frame
+DARKMATTER_TEXTURE = load_texture(DARKMATTER_TEXTURE_PATH)
+
+WALL_TEXTURE = load_texture(WALL_TEXTURE_PATH)
```
Comment (n0Oo0Oo0b) [inline pyweek36/constants.py:85]: I feel like loading textures shouldn't be done in `constants.py`, I think loading it when creating the class as a class attribute (like in `PlayerSprite`) would be better

Comment (n0Oo0Oo0b) [reply pyweek36/constants.py:85]: Or if we're going to load assets for tiles here, we may as well load all the assets here

**Review by n0Oo0Oo0b — APPROVED** [review]
it works I guess

**Pull request merged**